lib/screens/settings/quick_links.dart

```diff
@@ -148,9 +156,14 @@ class _QuickLinksScreenState extends State<QuickLinksScreen> {
 
   void addLink() {
     if (_formKey.currentState!.validate()) {
+      if (_labelEditingController.text.isEmpty) {
+        _labelEditingController.text = "My Link";
```

Comment:
No I mean we should fetch the metadata from the website like we did previously. For example if I enter google.com we can automatically figure out that it's titled Google.